commands/run.go

```diff
@@ -33,7 +73,7 @@ var dbConnectFlags = []cli.Flag{
 		Name:    "db",
 		EnvVars: []string{"LOTUS_DB"},
 		Value:   "",
-		Usage:   "A connection string for the postgres database, for example postgres://postgres:password@localhost:5432/postgres",
+		Usage:   "A connection string for the TimescaleDB database, for example postgres://postgres:password@localhost:5432/postgres",
```

Comment:
might be better to use `postgres://postgres:password@localhost:5432/postgres?sslmode=disable` as an example since it will work out of the box with our docker deployment in the Makefile.